dev/NavigationView/TestUI/Common/NavigationViewPage.xaml

```diff
@@ -26,6 +26,7 @@
                 DisplayModeChanged="NavView_DisplayModeChanged"
                 ItemInvoked="NavView_ItemInvoked"
                 SelectionChanged="NavView_SelectionChanged"
+                Header="TestHeader"
```

Comment:
what is this change for ?:)

Reply:
I guess I was using this to see where the header goes. Helpful, so I decided to keep it 😂